examples/apps/generic_client/eth_generic_client.py

```diff
@@ -0,0 +1,657 @@
+#! /usr/bin/env python3
```

Comment:
Make this file executable (`chmod +x  eth_generic_client.py` ), then`git commit` the permission change.